unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -934,19 +935,21 @@ loop e = do
               #numberedArgs .= fmap Name.toString patches
             FindShallowI pathArg -> do
               Cli.Env {codebase} <- ask
-              sbhLength <- liftIO (Codebase.branchHashLength codebase)
-              rootBranch <- Cli.getRootBranch
 
               pathArgAbs <- Cli.resolvePath' pathArg
               entries <- liftIO (Backend.lsAtPath codebase Nothing pathArgAbs)
               -- caching the result as an absolute path, for easier jumping around
               #numberedArgs .= fmap entryToHQString entries
-              let ppe =
-                    Backend.basicSuffixifiedNames
-                      sbhLength
-                      rootBranch
-                      (Backend.AllNames (Path.unabsolute pathArgAbs))
-              Cli.respond $ ListShallow ppe entries
+              getRoot <- atomically . STM.readTMVar <$> use #root
```

Comment:
This could use getRootBranch

Reply:
I need the 'getRoot' action in IO, but Cli doesn't have unliftio, so I can't use `getRootBranch` here.